identity/app/views/consentJourney.scala.html

```diff
@@ -68,6 +68,19 @@ <h3 class="manage-account__switch-title">&nbsp;</h3>
     )
 }
 
+@profilingConsentOptOutContent = {
+    @views.html.profile.profilingConsent(idUrlBuilder, idRequest, forms.privacyForm, user)(request, messages)
+}
+
+@profilingConsentOptOutStep = @{
+    ConsentStep(
+        name="profilingConsentOptOut",
+        title ="Using your data for marketing analysis",
```

Comment:
more of a general q, shouldn't we be pulling these titles from `idapi`? is this a potential future problem?

Reply:
AH! Sorry just realized this is a consent step, not the actual checkbox